packages/liquid_interpreter/liquid_interpreter.0.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis: "The interpreter for Liquid"
+description: """
+The interpreter for Liquid
+"""
+maintainer: "faerberbendev@protonmail.com"
+authors: ["Ben Faerber"]
+homepage: "https://github.com/benfaerber/liquid-ml"
+bug-reports: "https://github.com/benfaerber/liquid-ml/issues"
+dev-repo: "git+https://github.com/benfaerber/liquid-ml.git"
+license: "MIT"
+depends: [ 
+    "ocaml" { >= "4.11" }
+    "dune" { >= "2.5" } 
+    "base" { >= "0.13.1" }
+    "core" { >= "0.13.0" } 
+    "stdio" { >= "0.13.0" } 
+    "re2" { >= "0.13.0" }
+    "liquid_syntax" { = "0.1"} 
+    "liquid_parser" { = "0.1"}
+    "liquid_std" { = "0.1"}
```

Comment:
```suggestion
    "liquid_syntax" { = version} 
    "liquid_parser" { = version}
    "liquid_std" { = version}
```

It doesn't change much inside the opam-repository itself, but it's probably a good idea to use `version` upstream so you don't have to change it when you make a new release.

`version` is a variable (or a keyword I'm not sure) which holds (or expands to or whatever) the version of the package file that it is in. So `version` inside `packages/liquid_interpreter/liquid_interpreter.0.1/opam` is the same value as `"0.1"`, but `version` inside `packages/liquid_interpreter/liquid_interpreter.0.2/opam` will be the same value as `"0.2"`.